tests/frontends/test_pytorch_ops.py

```diff
@@ -311,6 +311,17 @@ def test_torch_conv2d__group3(inp, w, b):
     return torch.sum(value)
 
 
+@mt(
+    fwd_and_bwd(nn.Parameter(torch.Tensor(torch.randn(3, 5))),
+                nn.Parameter(torch.randint(5, (3,)),
+                             requires_grad=False)),
+    run(torch.randn(3, 5), torch.randint(5, (3,))),
+    backend=backend_no_relay
+)
```

Comment:
This should be:
```
@mt(
    fwd_and_bwd(nn.Parameter(torch.Tensor(torch.randn(3, 5))),
                nn.Parameter(torch.randint(5, (3,)),
                             requires_grad=False)),
)
```

Reply:
Done! I used `fwd_and_bwd_no_relay` instead of just `fwd_and_bwd`, because I got some compilation errors with relay, and I guess we don't need relay to work here as we are in Pytorch frontend, do we?

I also duplicated the test for each currently supported reduction (none, mean and sum)